doc/openshift-guide.md

```diff
@@ -8,23 +8,32 @@ To follow the tutorial steps below, you'll first need to address the following p
 
 ### 3scale account configuration
 
-These steps will guide you to finding information you will need later to complete the tutorial steps: your _3scale Admin URL_, your _Provider Key_, and the _Private Base URL_ of your API.
+These steps will guide you to finding information you will need later to complete the tutorial steps: your _3scale Admin URL_, the _Access Token_, and the _Private Base URL_ of your API.
 
 You should have a 3scale API provider account (you can sign up for a free trial <a href="https://www.3scale.net/signup/">here</a>) and have an API configured in the 3scale Admin Portal. If you do not have an API running and configured in your 3scale account, please follow the instructions in our <a href="https://support.3scale.net/guides/quickstart">Quickstart</a> to do so.
 
 Log in to your 3scale Admin Portal with the URL provided to you. It will look something like this: `https://MYDOMAIN-admin.3scale.net`
 
-Navigate to the **Account** tab (top-right). From the **Overview** sub-tab note down your _3scale Provider Key_, referred to in the UI as _API Key_. Later in this tutorial we refer to this as your *THREESCALE_PROVIDER_KEY*.
+#### Create an access token
```

Comment:
This can definitely be documented somewhere else, and just linked here, but I didn't want to do a total revamp, and just keeping the guide as complete as possible.